docs/Operators.md

```diff
@@ -30,6 +30,7 @@
 * <a href="#GlobalAveragePool">GlobalAveragePool</a>
 * <a href="#GlobalMaxPool">GlobalMaxPool</a>
 * <a href="#Greater">Greater</a>
+* <a href="#InstanceNorm">InstanceNorm</a>
```

Comment:
Shouldn't we call it InstantNormalization to match the same naming convention for other norm OP?